project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.3.6
+sbt.version=1.4.4
```

Comment:
I am not sure about this. We recently did some work to upgrade to v1.3.6 and I am not sure if this version is supprted by the platform.

Reply:
It does not harm. I already discussed this on Slack with team-platform. I'm already using sbt 1.4.x in my previous project.